parcel/parcel.jl

```diff
@@ -99,6 +99,15 @@ function parcel_model(dY, Y, p, t)
         end
         dqi_dt_new_depo = dN_act_dt_depo * 4 / 3 * FT(π) * r_nuc^3 * ρ_ice / ρ_air
     end
+    if "ActivityBasedDeposition" in ice_nucleation_modes
+        Δa_w = CMO.a_w_eT(tps, e, T) - CMO.a_w_ice(tps, T)
+        J_deposition = CMI_het.deposition_J(aerosol, Δa_w)
+        if "Monodisperse" in droplet_size_distribution
```

Comment:
Maybe we should add some error or at least warning that we don't support any other distribution type here?

Reply:
Yeah that's a good idea. Do you think it's ok to have a printed statement or should we have a hard `@assert`? I can also open an issue to add more size distributions to some of the freezing modes

Reply:
I think its ok to just add warnings, or whatever is the easiest. We will refactor parcel soon anyway.